client/js/screens/Campaigns/Campaigns.js

```diff
@@ -1,8 +1,131 @@
+import icebergMelt from '../../assets/Imagery/icebergmelt.png';
+import iceCream from '../../assets/Imagery/icecream.png';
+import imageGradient from '../../assets/buttons/buttonbackground.jpg';
+import playButton from '../../assets/buttons/videoplayicon.png';
 
-const Campaigns = props => {
-  return <Text>Campaigns Screen</Text>;
+const Campaigns = ({navigation, campaignData}) => {
+  return (
+    <SafeAreaView>
+      {/* Campaign Header Start */}
+      <View style={{flexGrow: 1, justifyContent: 'space-around'}}>
+        <View style={styles.campaignsHeader}>
+          <Image
+            style={styles.categoryImage}
+            source={require('../../assets/category/Charity.png')}
+          />
+
+          <Image
+            style={styles.xxxIcon}
+            source={require('../../assets/artwork/xgroup.png')}
+          />
+        </View>
+        {/* Campaign Header End */}
+
+        <ScrollView horizontal={true}>
+          <View style={styles.videoSection}>
+            {/* divider */}
+            <TouchableOpacity
+              onPress={() => {
+                Linking.openURL('https://www.youtube.com/watch?v=oxhYaiSnlAo');
+              }}
+              style={styles.touchableContainer}>
+              <ImageBackground source={iceCream} style={styles.categoryVideo}>
+                <View
+                  style={{
+                    flex: 1,
+                    justifyContent: 'flex-end',
+                    alignItems: 'flex-start',
+                  }}>
+                  <Image source={playButton} style={styles.videoPreview} />
+                </View>
+              </ImageBackground>
+            </TouchableOpacity>
+            {/* End of divider */}
+            {/* divider */}
+            <TouchableOpacity
+              onPress={() => {
+                Linking.openURL('https://www.youtube.com/watch?v=P3GagfbA2vo');
+              }}
+              style={styles.touchableContainer}>
+              <ImageBackground
+                source={icebergMelt}
+                style={styles.categoryVideo}>
+                <View
+                  style={{
+                    flex: 1,
+                    justifyContent: 'flex-end',
+                    alignItems: 'flex-start',
+                  }}>
+                  <Image source={playButton} style={styles.videoPreview} />
+                </View>
+              </ImageBackground>
+            </TouchableOpacity>
+            {/* End of divider */}
+            {/* divider */}
+            <TouchableOpacity
+              onPress={() => {
+                Linking.openURL('https://www.youtube.com/watch?v=KAJsdgTPJpU');
+              }}
+              style={styles.touchableContainer}>
+              <ImageBackground source={iceCream} style={styles.categoryVideo}>
+                <View
+                  style={{
+                    flex: 1,
+                    justifyContent: 'flex-end',
+                    alignItems: 'flex-start',
```

Comment:
Please move your inline comments to styles.js.

Reply:
Done